pkg/cmd/enclave_setup.go

```diff
@@ -118,6 +118,14 @@ func selectProject(projects []models.ProjectInfo, prevConfiguredProject string,
 		}
 	}
 
+	if len(projects) == 1 {
+		// the user is expecting to a prompt, so print a message instead
+		if promptUser {
+			utils.Log("Selected only available project")
```

Comment:
We should show the project name:

```
Selected only available project: Monolith
```

We could also bold the `Selected only available project:` part of the message.

I find it a little odd that I didn't see or select which project was selected. You can replicate this by having a workplace with a single project. I am using the `Doppler Demo` workpalce.

<img width="410" alt="Screen Shot 2020-05-08 at 7 23 16 PM" src="https://user-images.githubusercontent.com/1920007/81461536-63c60b00-9161-11ea-939f-8916d8377f00.png">

Reply:
> We should show the project name:

The reason we don't is because it's shown when we print the config.

Reply:
> We could also bold the Selected only available project: part of the message.

I think we should treat this like any other log message

> I find it a little odd that I didn't see or select which project was selected.

See the log message about it selecting the only available project

Reply:
Isn't the project id being shown, not the name of the project. I honestly found it a little jarring when I ran it and didn't know which project was auto-selected.

Reply:
I think this is the one use case where it makes sense to show it. I'll add the name in the message.

Reply:
Changes pushed, try out the latest version

Reply:
Testing!

Reply:
Nit: Can we change the color to a notice/warning color like orange and then bold the `Selected only available project:` part?

Reply:
The reason I'm avoiding putting emphasis on the lines is that I don't think we should need emphasis. If it's printed, it's important and the user should read it. If it's not important enough to be bolded, it shouldn't be printed at all.

Reply:
To be clear, I am not saying bold the entire line. Just the `Selected only available project:` part. 

For example the line could look like this:
**Selected only available project:** Partners (ep_d14e2b0a7b1)

I think this helps make it easier on the eye to see the message separated from the selected project. I know it's a nit but I think it would help.

Reply:
I follow what you're saying, I'm just not sure I agree. I pushed up a change with the bolded text so you can try it out.

Reply:
I like it!